src/frontend/src/components/user/changeInfo.tsx

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+
+const baseURL = 'http://127.0.0.1:5000/user';
+
+export function changeEmail(new_email: string) {
+   const req = {
+      is_email: true,
+      email: new_email,
+   };
+   const json = JSON.stringify(req);
+   axios({
+      method: 'POST',
+      url: `${baseURL}`,
+      data: json,
+      withCredentials: true,
+   })
+      .then((response) => {
+         console.log(response);
+      })
+      .catch((error) => {
+         if (error.response) {
+            console.log(error.response);
+            console.log(error.response.status);
+            console.log(error.response.headers);
+         }
+      });
+}
+
+export function changePhone(new_phone: string) {
+   const req = {
+      is_phone: true,
+      phone: new_phone,
+   };
+   const json = JSON.stringify(req);
+   axios({
+      method: 'POST',
+      url: `${baseURL}`,
+      data: json,
+      withCredentials: true,
+   })
+      .then((response) => {
+         console.log(response);
+      })
+      .catch((error) => {
+         if (error.response) {
+            console.log(error.response);
+            console.log(error.response.status);
+            console.log(error.response.headers);
+         }
+      });
+}
+
+export function logout() {
+   axios({
+      url: 'http://127.0.0.1:5000/logout',
+      withCredentials: true,
+   })
```

Comment:
does this not do anything

Reply:
We have a `logout` route in `app.py`, tested that and it works.

Reply:
ye that's fine I didn't understand it at first